src/accelerate/test_utils/scripts/test_ddp_comm_hook.py

```diff
@@ -69,6 +70,12 @@ def main():
         (DDPCommunicationHookType.BATCHED_POWER_SGD, DDPCommunicationHookType.FP16, {}),
         (DDPCommunicationHookType.BATCHED_POWER_SGD, DDPCommunicationHookType.BF16, {}),
     ]:
+        if is_hpu_available():
+            HPU_UNSUPPORTED_COMM_HOOKS = {DDPCommunicationHookType.FP16, DDPCommunicationHookType.BF16}
```

Comment:
It doesn't work with bf16?

Reply:
Yeah it seems like half precision comm hooks are only implemented on cuda, they failed with `require cuda` error.